src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Windows_NT;
-      netcoreapp-Unix;
+      netcoreapp2.0-Windows_NT;
+      netcoreapp2.0-Unix;
       netstandard;
```

Comment:
Why are some of the projects moving specifically to nc20 but most others are moving up to nc21?